src/PowerShellEditorServices.Host/CodeLens/PesterCodeLensProvider.cs

```diff
@@ -63,18 +74,53 @@ public PesterCodeLensProvider(EditorSession editorSession)
                     new ClientCommand(
                         "PowerShell.RunPesterTests",
                         "Debug tests",
-                        new object[] { scriptFile.ClientFilePath, true /* Run in debugger */, pesterSymbol.TestName })),
+                        new object[] { scriptFile.ClientFilePath, true /* Run in debugger */, pesterSymbol.TestName, describeBlockLineNumber })),
             };
 
             return codeLensResults;
         }
 
+        /// <summary>
+        /// Used to determine the value of <see cref="_pesterV4_6_0_OrHigherAvailable"/> as a background task.
+        /// </summary>
+        private async Task<bool> DeterminePesterVersion()
+        {
+            //if (System.Diagnostics.Debugger.IsAttached)
```

Comment:
nit: can you remove this commented out code

Reply:
Sorry, I forgot to take this out when I was attaching the debugger. This is definitely not a nit, this is an oops

Reply:
happens to the best of us 😊